src/onegov/org/utils.py

```diff
@@ -62,7 +62,7 @@
 # note that this tag should be safe - i.e. it may not accept '<' or the like
 # to avoid js script injection further down the line
 HASHTAG = re.compile(r'#\w{3,}')
-IMG_URLS = re.compile(r'<img src="(.*?)"')
+IMG_URLS = re.compile(r'<img.*?src="(.*?)"')
```

Comment:
```suggestion
IMG_URLS = re.compile(r'<img[^>]*?src="(.*?)"')
```
You can't allow all characters between img and src, but you can allow any except `>`.

Reply:
Right, thank you for noticing